### PR TITLE
[WIP] Content cta products

### PR DESCRIPTION
**Pull request by rshbhgrg**
Note: This PR is only for the purpose of experimental demos with content cta with multiple products and not intended to be the final design/direction.

### CODE REVIEW

File: core/frontend/helpers/content.js [hunk 1]
```diff
@@ -18,9 +18,26 @@ const createFrame = hbs.handlebars.createFrame;
 function restrictedCta(options) {
     options = options || {};
     options.data = options.data || {};
+
     _.merge(this, {
         accentColor: (options.data.site && options.data.site.accent_color)
     });
+    if (this.visibility.includes('product:')) {
+        const productSlugs = this.visibility.split(',').filter(qry => qry.startsWith('product:')).map(qry => qry.replace('product:', ''));
```
Comment (ErisDS) [inline core/frontend/helpers/content.js:26]: If visibility _is_ going to be used to store NQL strings, we should be using our language parser to parse them

File: core/frontend/helpers/tpl/content-cta.hbs [hunk 2]
```diff
@@ -7,6 +7,9 @@
         {{#has visibility="members"}}
             <h2>This post is for subscribers only</h2>
         {{/has}}
+        {{#if customVisibilityText}}
```
Comment (ErisDS) [inline core/frontend/helpers/tpl/content-cta.hbs:10]: If we're introducing this property as part of the theme API, has it passed design review?

File: core/frontend/helpers/content.js (hunk 1 above)
Comment (rishabhgrg) [reply core/frontend/helpers/content.js:26]: @ErisDS Ah yea good point! This was just a first version hacked out for demo purposes, but yeah didn't strike me to use language parser as the first thing to use here :)

File: core/frontend/helpers/tpl/content-cta.hbs (hunk 2 above)
Comment (rishabhgrg) [reply core/frontend/helpers/tpl/content-cta.hbs:10]: @ErisDS Its pending design review and don't expect the property to go out in current form for theme API. This PR was purely for demo purposes, sorry should have added a note in description clarifying(done now).

**Comment by rishabhgrg** [comment]
Closing for now as the PR was an early hack on how content cta can look with multiple products.

**Pull request closed without merging**